cpp/doxygen/examples/Ice/SSL/SchannelClientAuthenticationOptions.cpp

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+#include <Ice/Ice.h>
+
+#if defined(ICE_USE_SCHANNEL)
+void
+clientCertificateSelectionCallbackExample()
+{
+    //! [clientCertificateSelectionCallback]
+    PCCERT_CONTEXT clientCertificateChain{};
+    // Load the server certificate chain using Schannel APIs.
+    // ...
```

Comment:
I leave the loading of the certificates out of the examples to keep things relatively sort and to the point.